### ISSUE TITLE
better neighborhood bounds for Boston

### ISSUE THREAD

**Issue by slinkp** [openplans/openblock#62]
We're using Don K's very approximate rectangular boundaries.
Maybe use Zillow boundaries http://www.zillow.com/howto/api/neighborhood-boundaries.htm , although I'm not clear on the licensing terms - seems to require some attribution, but in what form?

**Comment by slinkp** [comment]
I once got a shapefile from the Boston Redevelopment Authority - let me see if I can find the right person there.

**Comment by slinkp** [comment]
Accompanying email:
Attached is a shapefile in State Plane Massachusetts mainland feet.  This file is only one way the BRA interprets planning districts in Boston.  We also have zip code areas and Zoning districts that have different boundaries. We need to be able to follow census tracts with our Planning districts boundaries.  Anyway, the file attached has 24 areas representing Planning Districts in Boston.  Please use the following disclaimer:

```
Disclaimer:  All reference to Planning Districts on any maps or digital 
data produced by the Office of Digital Cartography and GIS, Planning 
Department, BRA are for planning purposes only and should NOT be 
interpreted as official neighborhood boundaries for the City of 
Boston. Thank you.
```

This is hopefully a good starting point.  Do you have source for zipcode boundaries, or should I ask for that?

**Comment by slinkp** [comment]
I believe zipcode boundaries are available within TIGER already.  If those don't pan out I'll let you know.

**Comment by slinkp** [comment]
I added a file with a bunch of Boston zip code shapes extracted from TIGER.  It looks mainly correct visually, but it's not clear whether this this the set of zip codes Bostonians would expect to see :) 

The zip codes are: 

02108
02109
02110
02111
02113
02114
02115
02116
02118
02119
02120
02121
02122
02124
02125
02126
02127
02128
02129
02130
02131
02132
02134
02135
02136
02152
02203
02210
02215

**Comment by slinkp** [comment]
I'll check these out against what everyblock presents.

**Comment by slinkp** [comment]
Hmm they exclude 02203 and additionally include: 

02021
02026
02133
02138
02139
02141
02142
02143
02145
02151
02163
02186
02199
02201
02205
02222
02445
02446
02458
02459
02467
02472
02494

**Comment by slinkp** [comment]
(In [f4f2374cd0a02e29e22d7c049d531909c44bb423]) added disclaimer. closes #62

**Comment by slinkp** [comment]
Ticket imported from Trac:
 http://developer.openblockproject.org/ticket/62
Reported by: slinkp
